app/helpers/sorting_helper.rb

```diff
@@ -17,12 +17,17 @@ def sorting_item(dropdown, ransack_obj, field, dir)
                        })
   end
 
-  def sorting_url(ransack_obj, field, dir: nil)
-    url = if dir.nil?
-            sort_url(ransack_obj,
-                     field).to_s
+  def sorting_url(ransack_obj, field, dir: nil, with_params: true)
```

Comment:
**nitpick (blocking):** Can you rename `with_params` to `with_search_params`.

Reply:
Updated in 1e667007069ab6bb597cd8aabd077aad808cdbde